config/cc/gcc.in.2

```diff
@@ -94,6 +94,7 @@ config CC_GCC_SYSTEM_ZLIB
 config CC_GCC_CONFIG_TLS
     tristate
     prompt "Configure TLS (Thread Local Storage)"
+    default n if LIBC_bionic
```

Comment:
OK. I was thinking it should be that, but then wanted to leave a possibility of them adding TLS support later. I'll change this and force push over this.